### ISSUE TITLE
OverflowMenu doesn't allow distinction between icon's aria-label and alt text

### ISSUE THREAD

**Issue by rstrangfeld** [carbon-design-system/carbon#2507]
In the OverflowMenu, the way to provide an aria-label text for the visible icon (as required for accessibility) is to specify it as the "iconDescription" for the OverflowMenu. However, when you do this, the text you supply is also used as the "alt" string for the icon. This means that it is also displayed as a tooltip when a user hovers the mouse over the icon.

Normally there is a distinction between aria-label text and alt text, so that you can meet accessibility requirements without adding a visible tooltip for all users. That distinction is missing in OverflowMenu. 

A good way to provide it would be to have two separate properties in OverflowMenu, instead of using "iconDescription" for both purposes.
